src/main/java/io/zeebe/monitor/zeebe/ZeebeHazelcastService.java

```diff
@@ -13,7 +13,7 @@
 import javax.annotation.PreDestroy;
 import java.time.Duration;
 
-@Component
```

Comment:
Please make a configurable switch whether to use (enable) hazelcast or kafka

Reply:
It would be beneficial to make this configurable and try to open PR on the upstream project to get more open source community usage of Kafka + Simple Monitor